extensions/wikia/FlowTracking/scripts/initCreatePageTracking.js

```diff
@@ -49,12 +49,17 @@ require([
 			flowTracking.beginFlow(inputBoxFlow, {});
 		});
 
-		$( '#ca-ve-edit,  #ca-edit' ).click(function() {
+		$('#ca-edit').on('mousedown', function () {
```

Comment:
In VE `click` may stay because page isn't reloaded. Classic editor redirects you to new page, so we should send tracking earlier, before page is reloaded.